src/lib/zod/index.ts

```diff
@@ -261,11 +262,14 @@ export const HasuraUserSessionVariables = z
     'x-hasura-role': z.union([z.literal('user'), z.literal('superadmin')]),
     'x-hasura-address': zEthAddressOnly,
   })
-  .transform(vars => ({
-    hasuraProfileId: vars['x-hasura-user-id'],
-    hasuraRole: vars['x-hasura-role'],
-    hasuraAddress: vars['x-hasura-address'],
-  }));
+  .transform(vars => {
+    Sentry.setTag('action_user_id', vars['x-hasura-user-id']);
```

Comment:
I know this is sketchy but there is no other way to do this easily for every action with the way it is currently implemented - I might change it wrap the parse function and do it that way but this is ok for now